packages/retail-ui-extensions/src/extension-api/device-api/device-api.ts

```diff
@@ -0,0 +1,17 @@
+export interface DeviceApiContent {
+  /**
+   * The name of the device
+   */
+  name: string;
```

Comment:
What will this name entail? Like the operating system + model?

Reply:
I think it might be just `iPad` etc.

Reply:
Ah yeah, for model it looks like you'd need `getModel`